test/service/test_manifest.py

```diff
@@ -430,6 +443,7 @@ def test_manifest_zarr(self):
                     rows = [dict(file_crc32c=row['file_crc32c'],
                                  file_name=row['file_name'],
                                  file_uuid=row['file_uuid'],
+                                 file_drs_uri_path=furl(row['file_drs_uri']).path,
```

Comment:
We can assert more of the URL. Utilize

```python
>>> furl('drs://unpredictable/bar').set(host='known')
furl('drs://known/bar')
```

Reply:
> We can assert more of the URL.

Done.

I don't understand the purpose of modifying the host name of the `file_drs_uri` field in the manifest.